catt/controllers.py

```diff
@@ -497,15 +499,15 @@ def play_media_id(self, video_id):
     def play_playlist(self, playlist_id):
         raise NotImplementedError
 
-    def wait_for(self, states, invert=False, fail=False):
+    def wait_for(self, states, invert=False, fail=False, timeout=None):
```

Comment:
Can you add some types to these function definitions? It'll make things clearer.

Reply:
How would I annotate params with multiple valid types, without the ```typing``` module?

Reply:
Hmm, would this not work?:

```
def wait_for(self, states, invert:bool=False, fail:bool=False, timeout:Optional[int]=None):
```

Also, what's wrong with using the `typing` module?

Reply:
> Also, what's wrong with using the typing module?

It's py3.5+ only.

We could add it as a dependency for 3.4, as ```zeroconf``` is already doing this.

Reply:
Aw, I thought they added it from the start on 3. Yeah, that would be ideal, it's super useful. I can enable mypy checks afterwards.